app/src/main/kotlin/com/wire/android/ui/authentication/login/LoginViewModel.kt

```diff
@@ -73,6 +75,11 @@ class LoginViewModel @Inject constructor(
     private suspend fun navigateToConvScreen() =
         navigationManager.navigate(NavigationCommand(NavigationItem.Home.navigationRoute(), BackStackMode.CLEAR_WHOLE))
 
+    private fun clearErrorState() {
```

Comment:
why don't you update `clearLoginError()` and use it instead of separate fun?

Reply:
ooh didn't notice that once existed thx